app/components/pages/createPoll/styles.js

```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  CustomFullWidth: {
+    width: '100%',
+  },
+  buttonSize: {
```

Comment:
you're not using this class anywhere remote it

Reply:
deleted